src/mocks/handlers.ts

```diff
@@ -35,7 +37,7 @@ const pokemonListResponse = {
       url: "https://pokeapi.co/api/v2/pokemon/8/",
     },
   ],
-};
+} as PokemonListResponse;
```

Comment:
Evita quando possibile di usare `as`: https://dev.to/shakyshane/type-assertions-vs-declarations-the-dangers-of-as-in-typescript-1caf

Reply:
Chiaro. Lo metto come Assertion

Reply:
hai dimenticato di togliere l'as!